code/code/disc/disc_deikhan.cc

```diff
@@ -149,8 +149,6 @@ int TBeing::doLayHands(const char *arg)
     return FALSE;
   }
   one_argument(arg, name_buf, cElements(name_buf));
-  if (!*name_buf)
-    vict = this;
```

Comment:
Those two lines are dead code. The next line overwrites vict unconditionally, I would guess that get_char_room_vis() handles self or whatever.